packages/pn-commons/src/components/NotificationDetail/NotificationDetailTimelineStep.tsx

```diff
@@ -44,6 +44,7 @@ type Props = {
   historyButtonLabel?: string;
```

Comment:
there is another part where you must disable the download. check the code

Reply:
do it

Reply:
are you sure? i don't see differences